src/types/contributor.types.ts

```diff
@@ -3,18 +3,28 @@ enum ContributorRole {
   CREATOR = 'Creator',
 }
 
+enum ContributorType {
+  CONTRIBUTOR = 'Contributor',
+}
+
+enum ContributorIdentityType {
+  IDENTITY = 'Identity',
+}
```

Comment:
jeg har forstått det som at disse type-feltene ligger i alle objekter fra backend, og at de bare kan ha en fast verdi hver plass. 
Eksemplet til Jan på slack: https://unit-norge.slack.com/files/USCD9B6UF/FUWKPH5JN/v0.1.0

jeg tror det bare er ment som en backend-ting, for vi vet jo hva vi forventer i feks `.entityDescription` uansett. derfor har jeg bare droppet å spesifisere disse tidligere. alternativt skulle det kanskje vært en enum som hadde alle de ulike typene, men ser ikke helt når vi vil slå opp på det feltet uansett

Reply:
Grunnen til at jeg la det til var fordi det ikke gikk an å lagre uten de feltene. Det stod at det manglet en verdi for type både inni og utenfor identity.